Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor.cs

```diff
@@ -99,13 +108,35 @@ protected override void OnParametersSet()
 
 	protected bool ShouldBeExpanded => ExpandOnMatch && !ParentSidebar.Collapsed && this.childItems.Any(i => i.isMatch && i.ExpandOnMatch);
 	protected bool HasExpandableContent => (this.ChildContent is not null);
+	protected bool ShouldInitializePopper => HasExpandableContent && ParentSidebar.Collapsed;
 
 	protected override async Task OnAfterRenderAsync(bool firstRender)
 	{
+		if (firstRender)
+		{
+			await EnsureJsModule();
+		}
+
```

Comment:
Probably all the JS-related stuff should be conditional under `ShouldInitializePopper`?